### DISCUSSION TITLE
Key is invalid. You must supply a key in OpenSSH public key format.

### DISCUSSION THREAD

**Discussion by mariagzq** [community/community#48312] [New to GitHub]
Hi! I am trying to pair my SSH key with my GitHub account all I get is the following message: _Key is invalid. You must supply a key in OpenSSH public key format_   

I copied all the characters, tried with a new SSH key but the message is still appearing. What can I do? Thanks

**Comment by airtower-luna** [comment]
Without more details (what are you copying, etc.) the only thing I can say is: Double check you're copying the right thing. The public key should be exactly one line, starting with something like `ssh-rsa AAAA` (possibly a different type instead of `rsa`).

**Reply by mariagzq** [reply]
This is what I am copying

ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIHi0M2mg77oc2zj7wTgAMfN8d6pshsJbRaoDsw5RbISi
[email]

EDIT: Something was missing, now I could copy my key correctly. Thanks!

**Reply by airtower-luna** [reply]
I'm glad to hear you got it working! :slightly_smiling_face:
